base/linalg/generic.jl

```diff
@@ -417,48 +417,38 @@ istriu(x::Number) = true
 istril(x::Number) = true
 isdiag(x::Number) = true
 
-#TODO: lookup where all the documentation for linreg is coming from, including the example
-# mentioning PyPlot etc
 """
     linreg(x, y)
 
 Find the Ordinary Least Squares best linear fit between the numerical vectors `x` and `y`
 
 Inputs:
 
-- `x::AbstractVector{T <: Number}` - vector of independent variables
-- `y::AbstractVector{S <: Number}` - vector of dependent variables
+- `x::AbstractVector` - vector of independent variables
+- `y::AbstractVector` - vector of dependent variables
 
 Output:
 
-- `[b0, b1]` - the coefficients for the line of best fit such that yhat = b0 + b1*x
+- `(a, b)` - the coefficients for the line of best fit satisfying a + b*x
 
 See also:
 
 `\\`, `cov`, `std`, `mean`
 
 """
-function linreg{T <: Number, S <: Number}(x::AbstractVector{T}, y::AbstractVector{S})
+function linreg(x::AbstractVector, y::AbstractVector)
     # Least squares given
-    # Y = b0 + b1*X
+    # Y = a + b*X
```

Comment:
Out of curiosity, why change to `a` and `b`? Personally I've always found the subscripted beta notation for regression coefficients much clearer than `a` and `b`.